runtime/runtime-params-estimator/Cargo.toml

```diff
@@ -30,6 +30,8 @@ near-vm-runner = {path = "../../runtime/near-vm-runner" }
 node-runtime = { path = "../../runtime/runtime" }
 near-store = { path = "../../core/store" }
 near-primitives = { path = "../../core/primitives" }
+near-test-contracts = { path = "../../runtime/near-test-contracts" }
```

Comment:
Hm, I am surprised that estimator uses both `near-test-contracts` *and* it's own [test-contract](https://github.com/near/nearcore/blob/master/runtime/runtime-params-estimator/test-contract/build.sh). This is unrelated to the PR